homeassistant/components/climate/nest.py

```diff
@@ -174,13 +165,15 @@ def set_temperature(self, **kwargs):
     def set_operation_mode(self, operation_mode):
```

Comment:
I was able to track this down to code I changed in python-nest. However, I still don't understand how to exceptions would be received by hass when it's caused by something at runtime instead of bad code I wrote 😁